src/cluster.c

```diff
@@ -4410,6 +4410,43 @@ int getSlotOrReply(client *c, robj *o) {
     return (int) slot;
 }
 
+int getSlotStatus(client *c, unsigned char *slots, int del, int start_slot, int end_slot) {
+    int slot;
+    for(slot = start_slot; slot <= end_slot; slot++){
```

Comment:
minor style stuff
```suggestion
    for (slot = start_slot; slot <= end_slot; slot++) {
```

Reply:
It is very hard for humans to see stuff like this so I like to use a K&R formatter when possible.

Reply:
I really prefer to have a formatter. But it's hard to retroactively start that on a project.